jboss/container/wildfly/launch/deployment-scanner/module.yaml

```diff
@@ -5,3 +5,8 @@ description: Legacy os-eap-deployment-scanner script package.
 execute:
 - script: configure.sh
   user: '185'
+envs:
+  - name: "ADMIN_USERNAME"
```

Comment:
@jfdenise I think this should go in admin/module.yaml?

Reply:
Thanks will fix!